src/main/resources/data.sql

```diff
@@ -610,4 +623,507 @@ insert into user (id, first_name, middle_name, last_name, phone_number) values
+        ('146 Sporer Street','Apt. 342','Changbury','IA','37461','67a62b36-73b2-42d2-a2aa-bc2f1376d548'),
+        ('732 Kovacek Keys','Apt. 633','New Zandrafort','AL','47458','68715d85-50c7-4893-bc7d-8958c1f1df8d'),
+        ('8832 Zieme Shore','Apt. 185','Port Kimberliechester','OH','37680','68d139c0-ef47-41a0-86f1-7a3bbf4bdffd'),
+        ('99721 Schuppe Cliffs','Suite 213','New Shelaland','AZ','77056','692a66c8-dfe9-4ab6-bc27-1d3bed66d510'),
+        ('74122 Guillermina Expressway','Apt. 135','Dorlaburgh','NV','95551','69330179-a821-4b6f-afe1-1dab68d7a08e'),
+        ('742 Walsh Mission','Apt. 688','North Rayna','TN','17271','698331bb-5a70-4314-beba-643159d27964'),
+        ('4046 Candelaria Island','Apt. 427','Sheldonton','ND','33003','6c5ca21b-4776-4262-ba51-ac7af717279a'),
+        ('304 Cary Circles','Apt. 557','New Delmerfurt','ME','89497','6cd6441d-8fac-470f-a0dd-5facdc4b5e15'),
+        ('21287 Hegmann Freeway','Apt. 884','Barrettburgh','OK','06278','6d3c78d1-dcbd-4b93-8ab7-3efec8ed490c'),
+        ('469 Lang Extension','Apt. 660','Cassinshire','TN','57447','6d965d06-8c63-469c-8b9b-402da8fcd76b'),
+        ('13942 Shaunna Court','Suite 817','New Antionettemouth','MN','54799','6de6b80b-7a2a-48d0-9c6b-53936176e015'),
+        ('19353 Huel Oval','Apt. 340','South Timmy','DE','23397','6eda9af3-c024-41bf-8f33-4d34621dbcf3'),
+        ('2078 Effertz Port','Suite 907','West Jeni','DE','45428','6fb8577f-fd12-4422-bc9f-7b42cff62028'),
+        ('421 Sunday Underpass','Apt. 149','Mariettafort','ME','98711','6fd7083f-6aba-4fea-9978-9cb6c12e1efe'),
+        ('13467 Brekke Summit','Suite 838','Port Dion','MT','37822','706da23d-2bff-4e61-a105-e1407c7a73fc'),
+        ('2994 Will Junctions','Apt. 916','East Nevilleshire','MT','24642','70afd02e-3a9f-4aa9-b3e7-2ace702ee110'),
+        ('94771 Gale Village','Apt. 386','Wizaton','TN','20438','70b9ab5a-84d7-439d-8f5a-5f7e86d1b9a9'),
+        ('48892 Price Village','Suite 643','West Tobyshire','ID','52938','7102fe01-da3b-4709-96ba-2219799b3c7b'),
+        ('4879 Stoltenberg Ranch','Suite 936','Pourosfurt','HI','13194','719db19e-9f26-407f-bf99-a4f801c1733f'),
+        ('2959 Dierdre Shore','Apt. 885','New Davinachester','IL','90511','71e0b8e3-24d3-491f-af6a-672b2930e98f'),
+        ('540 Pamala Locks','Suite 255','Alishaburgh','MI','42777','7264af5a-6ffa-4c6b-ba05-479b69d7f1f4'),
+        ('203 Kuhlman Stream','Apt. 586','Trompchester','AL','34066','72e78bd6-43bd-43c8-b2db-277118f6120c'),
+        ('5284 Dianna Villages','Suite 950','Lake Carey','NJ','93395','735a5f74-8d84-438e-8b95-883230ccbb97'),
+        ('19323 Cassey Vista','Apt. 132','North Reginia','ME','31024','73883332-94b0-401f-bdb6-dbc7e66eff2e'),
+        ('342 Augustina Squares','Apt. 465','Lake Blondellchester','NM','34125','7390550f-2307-4c73-b1e3-99d11f58a5b5'),
+        ('544 Doyle Wells','Apt. 957','West Halley','TN','40077','73a3808a-bcae-4701-b0cc-fa3ca26e7c16');
+
+
+
```

Comment:
Like the samples to some extent.  In testing, they should all have a purpose.  e.g. ensure that all states work.